plugins/search/api-report.md

```diff
@@ -6,35 +6,28 @@
 /// <reference types="react" />
 
 import { BackstagePlugin } from '@backstage/core-plugin-api';
+import { DefaultResultListItemProps } from '@backstage/plugin-search-react';
 import { IconComponent } from '@backstage/core-plugin-api';
 import { InputBaseProps } from '@material-ui/core';
 import { ReactElement } from 'react';
 import { ReactNode } from 'react';
-import { ResultHighlight } from '@backstage/plugin-search-common';
 import { RouteRef } from '@backstage/core-plugin-api';
-import { SearchDocument } from '@backstage/plugin-search-common';
-import { SearchResult as SearchResult_2 } from '@backstage/plugin-search-common';
+import { SearchAutocompleteFilterProps as SearchAutocompleteFilterProps_2 } from '@backstage/plugin-search-react';
+import { SearchBarBaseProps as SearchBarBaseProps_2 } from '@backstage/plugin-search-react';
+import { SearchFilterComponentProps as SearchFilterComponentProps_2 } from '@backstage/plugin-search-react';
+import { SearchResultProps } from '@backstage/plugin-search-react';
+
+// @public @deprecated (undocumented)
+export const DefaultResultListItem: (
+  props: DefaultResultListItemProps,
+) => JSX.Element;
 
-// Warning: (ae-missing-release-tag) "DefaultResultListItem" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
```

Comment:
Now that all of these warnings are cleaned up, it'd be good to get this added to https://github.com/backstage/backstage/blob/master/scripts/api-extractor.ts#L212